packages/bottender/src/slack/SlackConnector.ts

```diff
@@ -151,10 +151,8 @@ export default class SlackConnector
     return this._client;
   }
 
-  getUniqueSessionKey(body: SlackRequestBody): string {
-    // FIXME: define types for every slack events
-    const rawEvent = this._getRawEventFromRequest(body) as any;
-
+  // FIXME: define types for every slack events
+  getChannelId(rawEvent: any): string | null {
```

Comment:
what will this method return if it's app home event?

Reply:
null